RequestService/RequestService.Repo/Repository.cs

```diff
@@ -2027,5 +2024,75 @@ public async Task<bool> DeleteRequest(int requestId, CancellationToken cancellat
                 return false;
             }            
         }
+
+        public async Task<IEnumerable<SupportActivityCount>> GetCompletedActivitiesCount(int? groupId)
+        {
+            groupId = groupId.HasValue ? groupId.Value : GENERIC_GROUPID;
```

Comment:
For the home page, we want to show the total of all requests - so there should be no ReferringGroupId in the where clause in that case.